ouroboros-network-protocols/testlib/Ouroboros/Network/Protocol/LocalStateQuery/Test.hs

```diff
@@ -266,32 +284,9 @@ instance ( Arbitrary point
          , Arbitrary result
          )
       => Arbitrary (AnyMessageAndAgencyWithResult block point query result) where
-      arbitrary = oneof
-        [ AnyMessageAndAgencyWithResult . getAnyMessageAndAgencyV7 <$> (arbitrary :: Gen (AnyMessageAndAgencyV7 block point query result))
-
-        , pure $ AnyMessageAndAgencyWithResult $ AnyMessageAndAgency (ClientAgency TokIdle)
-            (MsgAcquire Nothing)
-
-        , pure $ AnyMessageAndAgencyWithResult $ AnyMessageAndAgency (ClientAgency TokAcquired)
-            (MsgReAcquire Nothing)
-        ]
-
--- Newtype wrapper which generates only valid data for 'NodeToClientV7' protocol.
---
-newtype AnyMessageAndAgencyV7 block point query result = AnyMessageAndAgencyV7 {
```

Comment:
The lowest possible version is now 8. I'm removing this now because I suspect PR #3699 should have, and the code is simpler to maintain this way.